stk/ops/eltwise_ops.py

```diff
@@ -0,0 +1,31 @@
+from stk.matrix import Matrix
+
+def mul(a, b):
+    """Performs element-wise multiplication of matrices a and b.
+
+    It is the user's responsibility to make sure that a and b
+    follow the same matrix topology. This function assumes it is safe
+    to use the topoplogy of a.
+
+    Args:
+        a:
+           sparse stk.matrix.Matrix.
```

Comment:
You can put the description of each arg following the colon

`a: stk.Matrix`

Reply:
fixed!